src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -1871,117 +1871,117 @@
       "Size": 3588112
     },
     "lib/arm64-v8a/libmonodroid.so": {
-      "Size": 346992
+      "Size": 346848
     },
-    "META-INF/androidx.versionedparcelable_versionedparcelable.version": {
+    "META-INF/androidx.slidingpanelayout_slidingpanelayout.version": {
```

Comment:
This seems differently odd; why isn't this sorted?

(@jonpryor checks around…)

@radekdoulik : it looks like xamarin/xamarin-android#4613 never made it into radekdoulik/apkdiff!

Reply:
PR submitted to sort `apkdiff` output: https://github.com/radekdoulik/apkdiff/pull/4